src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -25,16 +25,17 @@ private RecipeHandler()
      * Initialize all recipes for minecolonies.
      *
      * @param enableInDevelopmentFeatures if we want development recipes.
-     * @param supplyChests                if we want supply chests or direct town hall crafting.
+     * @param supplyChests                if we want supply chests or direct
+     *                                    town hall crafting.
      */
     public static void init(final boolean enableInDevelopmentFeatures, final boolean supplyChests)
     {
         final String plankWood = "plankWood";
-        
+
         GameRegistry.addRecipe(new ItemStack(ModBlocks.blockConstructionTape, 16), "SWS", "S S", "S S",
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 16 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)